### ISSUE TITLE
Strange hlint message

### ISSUE THREAD

**Issue by aueelis** [gibiansky/IHaskell#510]
I discovered some strange message from hlint in the Notebook while playing with some code.

When I type

```
sn = (round $ sqrt $ 2545)
```

hlint discovers the unnecessary brackets and dollar sign. Though, the error message is a bit wrong:

```
Redundant bracket
Found:
(round sqrt sqrt  2545)
Why Not:
round sqrt sqrt  2545

Redundant $
Found:
sqrt $ 2545
Why Not:
sqrt 2545
```

Obviously, the part `$ sqrt $` gets replaced by `sqrt sqrt`. Passing this line directly to hlint produces the desired output.
##

**Comment by gibiansky** [comment]
Confirmed. Strange. I wonder if this has something to do with MathJax or something...

**Comment by vaibhavsagar** [comment]
Closed by #1111.
